module/VuFindTheme/src/VuFindTheme/Initializer.php

```diff
@@ -341,14 +341,6 @@ protected function setUpThemes($themes)
             $resources->setGenerator($this->config->generator);
         }
 
-        $lessActive = false;
```

Comment:
This code was orphaned when PHP LESS compilation was removed in release 9.0 -- see #2586.